autoPyTorch/pipeline/components/preprocessing/tabular_preprocessing/encoding/OneHotEncoder.py

```diff
@@ -2,27 +2,24 @@
 
 import numpy as np
 
-from sklearn.compose import make_column_transformer
 from sklearn.preprocessing import OneHotEncoder as OHE
 
-from autoPyTorch.pipeline.components.preprocessing.encoding.base_encoder import BaseEncoder
+from autoPyTorch.pipeline.components.preprocessing.tabular_preprocessing.encoding.base_encoder import BaseEncoder
 
 
 class OneHotEncoder(BaseEncoder):
     """
     Encode categorical features as a one-hot numerical array
     """
     def __init__(self, random_state: Optional[Union[np.random.RandomState, int]] = None):
```

Comment:
I see this repeated in the NoEncoder and here. Do you think it makes sense to move it to the base class?